packages/suite/src/support/extraDependencies.ts

```diff
@@ -37,6 +37,8 @@ const connectInitSettings = {
 export const extraDependencies: ExtraDependencies = {
     thunks: {
         notificationsAddEvent: addEvent,
+        cardanoValidatePendingTxOnBlock: validatePendingTxOnBlock,
+        cardanoFetchTrezorPools: cardanoStakingActions.fetchTrezorPools,
```

Comment:
**import * as cardanoStakingActions from '@wallet-actions/cardanoStakingActions';**
import * as walletSettingsActions from '@settings-actions/walletSettingsActions';
import * as blockchainActions from '@wallet-actions/blockchainActions';
import { selectIsPendingTransportEvent } from '../reducers/suite/deviceReducer';
import { selectIsPendingTransportEvent } from '@suite-reducers/deviceReducer';
import * as suiteActions from '../actions/suite/suiteActions';
import { isWeb } from '@suite-utils/env';
import { resolveStaticPath } from '@trezor/utils';
**import { validatePendingTxOnBlock } from '@wallet-actions/cardanoStakingActions';**

Merge these two in bold to a single import:

`import * as cardanoStakingActions from '@wallet-actions/cardanoStakingActions';`